scripting/get5.sp

```diff
@@ -417,6 +426,8 @@ public void OnPluginStart() {
   g_OnSeriesInit = CreateGlobalForward("Get5_OnSeriesInit", ET_Ignore);
   g_OnSeriesResult =
       CreateGlobalForward("Get5_OnSeriesResult", ET_Ignore, Param_Cell, Param_Cell, Param_Cell);
+  g_OnSeriesResultWithoutWinners =
```

Comment:
If you add a forward you should update the include (scripting/include/get5.inc). 

That said, I don't think this needs a new forward. I think just calling the existing ``Get5_OnSeriesResult`` with the 1st argument (seriesWinner) as MatchTeam_TeamNone.

Reply:
Hi, sorry for the (giant) delay. If the Forward isn't  necessary, can you help me to remove it? I don't know exactly what I have to remove.

I think that the code might be different by now, considering that this PR is from July. Can you help me to merge the things out and make a valid and updated PR?

Thanks.